### PR TITLE
[5206] Optimize caching headers for static resources and add cache busters where needed

### PR DESCRIPTION
**Pull request by jvega190**
https://github.com/craftercms/craftercms/issues/5206

### CODE REVIEW

File: static-assets/components/cstudio-common/common-api.js [hunk 1]
```diff
@@ -2982,8 +2980,7 @@ var nodeOpen = false,
       _formatURL: function (url) {
         return CStudioAuthoring.StringUtils.keyFormat(url, {
           site: CStudioAuthoringContext.site,
-          base: CStudioAuthoringContext.baseUri,
-          version: CStudioAuthoring.UIBuildId
```
Comment (rart) [inline static-assets/components/cstudio-common/common-api.js:2986]: Check version isn't getting sent and ending up in the url unreplaced

File: static-assets/components/cstudio-common/common-api.js [hunk 2]
```diff
@@ -82,8 +82,6 @@ var nodeOpen = false,
       processing: false,
       compConfProcessing: false,
 
-      UIBuildId: window.UIBuildId,
```
Comment (rart) [inline static-assets/components/cstudio-common/common-api.js:85]: Check for other usages of uibuildid nd if possible remove the global for good

**Review by rart — APPROVED** [review]

File: static-assets/components/cstudio-common/common-api.js (hunk 2 above)
Comment (jvega190) [reply static-assets/components/cstudio-common/common-api.js:85]: There are no more usages, and the global was removed in `head.ftl`

File: templates/web/common/page-fragments/head.ftl [hunk 3]
```diff
@@ -18,54 +18,50 @@
   <meta http-equiv="refresh" content="0;URL='/studio/user-dashboard/#/sites/all'" />
 <#else>
 
-  <script>
```
Comment (jvega190) [inline templates/web/common/page-fragments/head.ftl:21]: @rart here

File: templates/web/common/page-fragments/head.ftl [hunk 4]
```diff
@@ -18,54 +18,50 @@
...
-    window.UIBuildId = "${UIBuildId!.now?string('Mddyyyy')}";
```
Comment (jvega190) [inline templates/web/common/page-fragments/head.ftl:22]: @rart here

**Review by rart — APPROVED** [review]

**Pull request merged**